4-css-grid/grid.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grid Exercise</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>Grid Exercise</h1>
+    <div class="grid-container">
+        <nav class="block-a"></nav>
+        <section class="block-b"></section>
+        <section class="block-c"></section>
+        <aside class="block-d"></aside>
```

Comment:
Keep similar elements with the same tag.

```suggestion
        <section class="block-d"></section>
```